lib/WebpackOptionsDefaulter.js

```diff
@@ -56,6 +56,10 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			{
 				test: /\.wasm$/i,
 				type: "webassembly/experimental"
+			},
+			{
+				test: /\.html$/,
```

Comment:
The other RegExps have the `i` flag but this one doesn't. Oversight?

Reply:
Likely :)